CMakeLists.txt

```diff
@@ -73,6 +73,7 @@ endif()
 ###### OpenVDB Build/Component Options
 
 include(CMakeDependentOption)
+include(GNUInstallDirs)
```

Comment:
Does this need to be included in every lists? Can we include it once in the root (here)?

Reply:
The multiple inclusion can be considered a remnant from back when there were separate `openvdb_houdini` and `openvdb` repos, and now they are combined.

`CMakeLists.txt` specifies a different `project(...)` in each subdir, which to me means, "these are in a single repo, but really they're different projects and they should be able to be built independently".

That was my assumption -- that they should be able to build independently.  That assumption led me to add the include to each component subdir.   Let me know if that's not important and I'll go ahead and include it from the top-level only instead.

Reply:
> CMakeLists.txt specifies a different project(...) in each subdir, which to me means, "these are in a single repo, but really they're different projects and they should be able to be built independently".

Right, that's a fair enough assumption - unfortunately most of the CMakeLists have been put together under the assumption you're running cmake from the root repo and using the available configuration controls to isolate individual projects. I can foresee a number of issues with the current structure were you to try and use them as you describe. That being said it's probably not a bad idea to try and support going forward. I'm happy to have them included as you've done so